### ISSUE TITLE
White flash on page transitions

### ISSUE THREAD

**Issue by 32u2** [h-enk/doks#167]
## Summary

Added the following script on top of layouts\partials\header\header.html to:
- avoid white flash between page transitions in the dark mode, and
- set dark theme as default

### Basic example

```
<script>
  if (localStorage.getItem('theme') === 'dark' || localStorage.getItem('theme') === null) {
    document.body.classList.add('dark');
  }
</script>
```

### Motivation

Not sending PR as this is just a quick hack I needed.

**Comment by h-enk** [comment]
See also: https://github.com/h-enk/doks/discussions/64

**Comment by h-enk** [comment]
Fixed w/ Doks [v0.3.4](https://github.com/h-enk/doks/releases/tag/v0.3.4)
